integrations/event-handler/cli.go

```diff
@@ -214,7 +214,7 @@ type CLI struct {
 	Config kong.ConfigFlag `help:"Path to TOML configuration file" optional:"true" short:"c" type:"existingfile" env:"FDFWD_CONFIG"`
 
 	// Debug is a debug logging mode flag
-	Debug bool `help:"Debug logging" short:"d"`
+	Debug bool `help:"Debug logging" short:"d" env:"FDFWD_DEBUG"`
```

Comment:
Why not use `TELEPORT_DEBUG` like all of our other binaries?

Reply:
This matches with the other environment variable approaches. https://github.com/gravitational/teleport/tree/master/integrations/event-handler